profile-models/src/main/java/com/telecominfraproject/wlan/profile/rf/models/RfElementConfiguration.java

```diff
@@ -19,12 +24,12 @@
 	
     private static final Logger LOG = LoggerFactory.getLogger(RfElementConfiguration.class);
     
-//  private final static int MIN_TWOFOURG_RADIO_CELL_SIZE = -80;
-//	private final static int MIN_FIVEG_RADIO_CELL_SIZE = -80;
-//	private final static int MIN_FIVEGUPPER_RADIO_CELL_SIZE = -80;
-//	private final static int MIN_FIVEGLOWER_RADIO_CELL_SIZE = -80;
-//	public final static int DEFAULT_RX_CELL_SIZE_DB = -90;
-//	public final static int DEFAULT_EIRP_TX_POWER = 18;
+    private final static int MIN_TWOFOURG_RADIO_CELL_SIZE = -80;
```

Comment:
Can the MIN constants be moved into a map per radioType. We will only have more of them (eg 6G)

Reply:
I can make that adjustment